invoices/invoiceregistry.go

```diff
@@ -1043,3 +1141,309 @@ func (i *InvoiceRegistry) HodlUnsubscribeAll(subscriber chan<- interface{}) {
 
 	delete(i.hodlReverseSubscriptions, subscriber)
 }
+
+// updateResult is the result of the invoice update call.
```

Comment:
this whole section could probably be moved to a separate file. perhaps `update.go`? `updateinvoice.go` would also work, but maybe redundant with the package name.

Reply:
done